code/modules/reagents/chemistry/reagents/drink_reagents.dm

```diff
@@ -437,6 +437,17 @@
 	M.adjust_bodytemperature(-5 * TEMPERATURE_DAMAGE_COEFFICIENT, BODYTEMP_NORMAL)
 	..()
 
+/datum/reagent/consumable/strongebow
```

Comment:
So, if the soda can of Strong-eBow just contains apple juice and uranium, why are you adding an unused reagent with a similar name?